form.js

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable no-unused-vars */
+
+// Mobile
+
+const form = document.querySelector('.form-element');
+const message = 'Email must contain only <br> lower case characters';
+
+function showMessage(input, message) {
+  const msg = document.querySelector('.validation-msg');
+  msg.innerHTML = `<p class='error-msg'>${message}</p>`;
+}
+
+function validateEmail(input) {
+  const value = input.toLowerCase();
+  if (value.localeCompare(input) === 0) {
+    return true;
+  }
+  return false;
+}
+
+form.addEventListener('submit', (event) => {
+  const msg = document.querySelector('.validation-msg');
+  event.preventDefault();
+  const input = form.elements[1];
+  const emailValid = validateEmail(input.value);
+  if (emailValid) {
+    msg.remove();
+    form.submit();
+    form.reset();
+  } else {
+    showMessage(input, message, false);
+  }
+});
+
+// Desktop
```

Comment:
- [ ] Congratulations I can see you are checking the validation before submitting the form but what I would really recommend you to do is to change the way you can display the error to the visitor. One of the requirements says that The error message should show up close to the "submit" button and not overwrite the input as you have done. check the screen video below 👇  👇  on how I am getting your implementation.


https://user-images.githubusercontent.com/65068771/147267295-26c2f822-6d54-4576-8c11-86469dfa6bb2.mp4


- [ ] kindly I would recommend also working on your style by following the design of the template. check the difference here down 👇  👇 

Your design  |  |
-------------|  - |
![image](https://user-images.githubusercontent.com/65068771/147268035-6f700fd6-17e0-4fdc-80da-625648b4a649.png) |  |
The original design |  |
![image](https://user-images.githubusercontent.com/65068771/147268145-8093b465-ccd9-43be-a6e7-3a32d96e658e.png) |  |

Reply:
Hi, Anderson. Thank you for the suggestion. 
But I should ask you to clarify some details. 
In the original design, we have no error message. 
Do you mean text inside textarea? 
If so, would it be good enough to place a message in textarea?
Thank you in advance.

Reply:
Sorry, Anderson, I can’t wait any longer for an answer. My week is in danger of failure. I put the message in the textarea over the button. Hope for a positive answer.